src/renderer/components/+catalog/catalog.tsx

```diff
@@ -250,6 +250,9 @@ class NonInjectedCatalog extends React.Component<Props & Dependencies> {
       ? `catalog-items-${activeCategory.metadata.name.replace(" ", "")}`
       : "catalog-items";
 
+    const noItemsMessage = activeCategory?.metadata.name == "Dev Clusters" ? "There are no Dev Clusters in this space"
```

Comment:
Can we add this to the `CatalogCategory` (same as `icon` and `name`) metadata? So the category could define it's own `noItemsMessage`?

@panuhorsmalahti @Nokel81

Reply:
Oh yes, this most certainly should not be special cased like this.